index.html

```diff
@@ -282,10 +282,17 @@ <h1 class="title contact-title">
         </p>
         <div class="contact-form">
             <form action="https://formspree.io/f/xpzebrvw" method="POST">
+<<<<<<< HEAD
+                <input type="text" name="name" maxlength="30" placeholder="Lorem Ipsum" required>
+                <input type="email" name="email" placeholder="loremipsum@gmail.com" required>
+                <textarea name="message" cols="30" rows="10" maxlength="500" placeholder="Write your message here..." required></textarea>
+                <button class="project-link" type="submit">Get in touch</button>
+=======
```

Comment:
As a result of the resolved conflict, you are having a duplicate of the form inputs on the UI and hence when the forms are being submitted, an error is given back. Kindly resolve this in order to make the form to function correctly.

Example of an error

<img width="328" alt="fsfs" src="https://user-images.githubusercontent.com/84930355/236190946-753df122-85fd-4f40-8b2a-9d43c4beb0d4.PNG">

Ensure to remove the below texts that came from the conflict:
- <<<<<<< HEAD
- =======
- '>>>>>>> main'